R/tm_a_regression.R

```diff
@@ -448,9 +448,14 @@ srv_a_regression <- function(id,
       )
     })
 
+    qenv <- teal.code::eval_code(
+      data(),
+      "library('ggplot2');"
```

Comment:
With Pawel suggestion libraries should have `"` inside
```suggestion
      'library("ggplot2")' # nolint quotes
```